R/FilterStates.R

```diff
@@ -123,7 +123,7 @@ FilterStates <- R6::R6Class( # nolint
     #'
     #' @return `call` or `NULL`
     #'
-    get_call = function() {
+    get_call = function(sid = integer(0)) {
```

Comment:
I'd be explicit with a name with get_call: `sid -> excluded_filterstate_ids` or something like that

Reply:
I've chosen this name as `excluded_id` or similar could be quickly changed. We have a straight way now to implement hierarchical filter. Let's assume we have 4 filters and we'd like to update counts of the filter-3:

- 1
- 2
- 3**
- 4

```
# 1. my branch here - all filters except this one
states <- private$state_list[where sid != sid]

# 2. future - hierarchical
states <- private$state_list[where sid < sid]
```

So the phrase `excluded` doesn't really have anything in common with (2). Other proposition?